docs/install/cf-proxy.md

```diff
@@ -19,7 +19,7 @@ Clone the [service-broker-proxy-cf](https://github.com/Peripli/service-broker-pr
 git clone https://github.com/Peripli/service-broker-proxy-cf.git && cd service-broker-proxy-cf
 ```
 
-**Note:** Do not use `go get`. Instead use git to clone the repository.
+**Note:** Do not use `go get`. Instead use git to clone the repository, but within `$GOPATH/src` to allow `dep ensure`.
```

Comment:
I can rework this, but honestly I didn't fully understand it...

* It also worked in $GOPATH/src/service-broker-proxy-cf
* Why not simply use `go get`? Would it has the same outcome?
* Why is `dep ensure` needed in the first place? Shouldn't it be already be vendored?

Reply:
* In the docs we would like to recommend what Go recommends whereever possible - so here we follow Go's recommendation regarding the project file structure in the go workspace.
* So there are some situations where `go get` fails. If i remember correctly here is one - go get the service-manager repo and dep ensure it. Now go get the cf-proxy one. Go get would actually finish cloning successfully but the go install part would fail and this brings confusion - so in order to avoid the confusion we just recommend to do manual cloning. 
* We currently do not submit the vendor folder in most of the repositories (we submit the .lock and .toml) and running dep ensure --vendor-only would download the dependencies specified in the .lock and create the vedor directory